omod/src/main/java/org/openmrs/module/openconceptlab/web/rest/resources/ImportResource.java

```diff
@@ -173,4 +178,27 @@ private UpdateScheduler getUpdateScheduler() {
     private static ImportService getImportService() {
         return Context.getService(ImportService.class);
     }
+
+    @Override
+    public Object upload(MultipartFile multipartFile, RequestContext requestContext) throws ResponseException, IOException {
+
+        ImportService importService = getImportService();
+
+        Importer importer = Context.getRegisteredComponent("openconceptlab.importer", Importer.class);
+        importer.setImportService(importService);
+
+        File file = new File(multipartFile.getOriginalFilename());
+        multipartFile.transferTo(file);
```

Comment:
Did you think of what happens to a file once it is processed? We don't want to keep them forever. It would be easier not to save the file to disk, rather keep it in memory. No need to worry about discarding it that way. The uploaded file should easily fit in memory CIEL ~50MB.